### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by cody151** [sergeyCodenameOne/UberEatsClone#17]
Android studio and IDEA by intellij used but it keeps saying module not specified, despite resyncing and clearing cache.

**Comment by shai-almog** [comment]
Make sure you're using JDK 11.

**Comment by cody151** [comment]
> Make sure you're using JDK 11.

I don't believe this to be the issue, I have JDK already installed and the modules dropdown does not show any modules at all in the list

**Comment by shai-almog** [comment]
Again. JDK 11. Not newer. See https://www.youtube.com/watch?v=BzySM_NpcEI
